lib/dealership.rb

```diff
@@ -0,0 +1,62 @@
+class Dealership
+
+  attr_reader :name,
+              :address,
+              :inventory,
+              :inventory_count
+
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+    @inventory_count = 0
+  end
+
+  def add_car(car)
+    @inventory << car
+    @inventory_count += 1
+  end
+
+  def has_inventory?
+    @inventory.count > 0
```

Comment:
I would stick with either using @inventory.count or @inventory_count throughout this class.